.github/workflows/test-integrations.yml

```diff
@@ -616,7 +616,30 @@ jobs:
         run: chmod +x consul
       - name: Build consul:local image
         run: docker build -t ${{ env.CONSUL_LATEST_IMAGE_NAME }}:local -f ./build-support/docker/Consul-Dev.dockerfile .
-      - name: Peering commonTopo Integration Tests
+      - name: Peering Integration Tests - Shared Topology Tests
+        env:
+          COMMON_TOPOLOGY: "true"
+        run: |
+          mkdir -p "${{ env.TEST_RESULTS_DIR }}"
+          cd ./test-integ/peering_commontopo
+          docker run --rm ${{ env.CONSUL_LATEST_IMAGE_NAME }}:local consul version
+          go run gotest.tools/gotestsum@v${{env.GOTESTSUM_VERSION}} \
+            --raw-command \
+            --format=short-verbose \
+            --debug \
+            --packages="./..." \
+            -- \
+            go test \
+            -tags "${{ env.GOTAGS }}" \
+            -timeout=30m \
+            -run '^TestAllCommonTopoSuites' \
```

Comment:
Hm, the goal of `-no-reuse-common-topo` was to always run all the tests, but just in different ways, without having to do a `-run` selector. So `go test -no-reuse-common-topo .` would run all tests, and none of them sharing topo; `go test .` would run all tests, except that the shareable ones (AC1, 2, 3, 5.1) would share a topology. Looks like how you've implemented it here, `COMMON_TOPOLOGY=true go test .` would run all tests, but the shared topo tests would run *again* in non-shared mode. I don't think that's desireable.

Reply:
Ah, I noticed this and fixed it.
```
go test -timeout=10m -v .
=== RUN   TestAC1Basic
--- PASS: TestAC1Basic (0.00s)
=== RUN   TestAC2DiscoChain
--- PASS: TestAC2DiscoChain (0.00s)
=== RUN   TestAC3SvcDefaults
--- PASS: TestAC3SvcDefaults (0.00s)
=== RUN   TestAC4ProxyDefaults
--- PASS: TestAC4ProxyDefaults (0.00s)
=== RUN   TestAC5ServiceMeshDisabledSuite
--- PASS: TestAC5ServiceMeshDisabledSuite (0.00s)
=== RUN   TestAC5PreparedQueryFailoverSuite
--- PASS: TestAC5PreparedQueryFailoverSuite (0.00s)
=== RUN   TestAC6Failovers
--- PASS: TestAC6Failovers (0.00s)
=== RUN   TestRotateGW
--- PASS: TestRotateGW (0.00s)
=== RUN   TestAC7_2RotateLeader
--- PASS: TestAC7_2RotateLeader (0.00s)
=== RUN   TestAllCommonTopoSuites
    main_test.go:28: Skipping: run "go test -commontopo" to run this test suite
--- SKIP: TestAllCommonTopoSuites (0.00s)```

```
go test -timeout=10m -v -commontopo . 
=== RUN   TestAC1Basic
    ac1_basic_test.go:18: Skipping: duplicate test run. See "TestAllCommonTopoSuites"
--- SKIP: TestAC1Basic (0.00s)
=== RUN   TestAC2DiscoChain
    ac2_disco_chain_test.go:19: Skipping: duplicate test run. See "TestAllCommonTopoSuites"
--- SKIP: TestAC2DiscoChain (0.00s)
=== RUN   TestAC3SvcDefaults
    ac3_service_defaults_upstream_test.go:27: Skipping: duplicate test run. See "TestAllCommonTopoSuites"
--- SKIP: TestAC3SvcDefaults (0.00s)
=== RUN   TestAC4ProxyDefaults
    ac4_proxy_defaults_test.go:33: Skipping: duplicate test run. See "TestAllCommonTopoSuites"
--- SKIP: TestAC4ProxyDefaults (0.00s)
=== RUN   TestAC5ServiceMeshDisabledSuite
    ac5_1_test.go:31: Skipping: duplicate test run. See "TestAllCommonTopoSuites"
--- SKIP: TestAC5ServiceMeshDisabledSuite (0.00s)
=== RUN   TestAC5PreparedQueryFailoverSuite
--- PASS: TestAC5PreparedQueryFailoverSuite (0.00s)
=== RUN   TestAC6Failovers
--- PASS: TestAC6Failovers (0.00s)
=== RUN   TestRotateGW
--- PASS: TestRotateGW (0.00s)
=== RUN   TestAC7_2RotateLeader
--- PASS: TestAC7_2RotateLeader (0.00s)
=== RUN   TestAllCommonTopoSuites
--- PASS: TestAllCommonTopoSuites (0.00s)
```